PortScanner.py

```diff
@@ -39,10 +36,8 @@ def main():
 	tgtHost = options.tgtHost
 	tgtPorts = str(options.tgtPort).split(',') 
 	if (tgtHost == None) | (tgtPorts[0] == None):
-		print '[-] You must specify a target host and port[s].'
-		exit(0)
+	   print '[-] You must specify a target host and port[s].'
+	exit(0)

```

Comment:
they both should be on same level...exit function call is also the part of "if" block